cmd/record.go

```diff
@@ -19,7 +19,7 @@ func NewCmdRecord(logger *zap.Logger) *Record {
 	recorder := record.NewRecorder(logger)
 	return &Record{
 		recorder: recorder,
-		logger:   logger,
+		Logger:   logger,
```

Comment:
If this is not used by the external packages, then it can be unexported.

Reply:
Done